core/frontend/src/tile/ImdlReader.ts

```diff
@@ -41,10 +41,10 @@ export interface ImdlReaderResult extends IModelTileContent {
  * @param iModel The iModel with which the graphics are associated.
  * @param modelId The Id of the [[GeometricModelState]] with which the graphics are associated. Can be an invalid Id.
  * @param is3d True if the graphics are 3d.
- * @param options Options customizing how [Feature]($common)s within the graphic can be resymbolized.
+ * @param options Options customizing how [Feature]($common)s within the graphic can be resymbolized; or false if you don't want to produce a batch.

```

Comment:
@bbastings you had a use case for this at some point; I think you instead set the feature to non-locatable? This would be more efficient.